camel-core/src/main/java/org/apache/camel/ComponentVerifier.java

```diff
@@ -412,6 +411,12 @@ public String toString() {
         }
     }
 
+    class ExceptionAttributeImpl extends ErrorAttribute implements VerificationError.ExceptionAttribute {
+        private ExceptionAttributeImpl(String name) {
+            super(name);
+        }
+    }
+
```

Comment:
Or we could leave it in this class but then leak `ExceptionAttributeImpl` (which no one can instantiate because of a private constructor) (and it is already big)